### ISSUE TITLE
tmpParts2 not assigned / activeaddressbooks can't be loaded - login not possible

### ISSUE THREAD

**Issue by minami-o** [equaeghe/infcloud#24]
Hi there,
**TL; DR:**
Replace erroneous index 2 in the last square bracket (after the split function) of [this line](https://github.com/equaeghe/infcloud/blob/master/main.js#L1117) with correct index 1 to prevent the corruption of the user settings which would prevent any further logon.

**Long story:**
First off, thanks a lot for this very nice piece of software you have built!
I just installed it on my vps as a replacement for the long not updated agendav, and I was surprised of the progress made since 2013.
I think this issue is really a coding error and doesn't depend of the setup, but my setup is the following:
- ubuntu server 14.04.2 LTS
- mariadb 5.5.44
- apache 2.4.7
- php 5.5.9-1ubuntu4.11
- sabredav 2.1.3 (with the hack you provide to save settings in the principals properties)
- tested on Firefox 38.0.5 and Chromium 44.0.2395.0 (64-bit)

My issue was the following:
- first logon is fine
- first logoff seems fine
- second logon: fails in [the middle of the loading phase](https://github.com/equaeghe/infcloud/blob/master/interface.js#L5756), with a message depending on the browser, but always saying that there is something wrong with the saved setting for activeaddressbooks (like "unable to evaluate tmpParts2 which is not assigned" or so, tmpParts2 being a substring of the mentioned setting)

From this point, the browser is stucked and the profile corrupt, there is no way to log in again unless you clear the wrong setting directly in the database.
I noticed that, indeed, the form of the members of the setting called "activeaddressbookcollections" in the database was "https://undefined", which was weird...
I investigated further and found that [one rule](https://github.com/equaeghe/infcloud/blob/master/main.js#L1117) in the saving of the mentioned setting at logout is not correct: the index 2 it recovers does not exist in the split function, which can only have two members, the last one with index 1.
With this corrected, the profile is correctly saved and the profile safe!

**Comment by equaeghe** [comment]
This is a personal fork of infcloud. Please contact the developers directly: https://www.inf-it.com/open-source/clients/infcloud/
